### PR TITLE
24 return random name

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Added the switch cases for all Races and Genders.

### CODE REVIEW

**Review by KingFruit85 — APPROVED** [review]
👏

**Review by KingFruit85 — COMMENTED** [review]
👏

**Review by KingFruit85 — APPROVED** [review]
👏

File: names.js [hunk 1]
```diff
@@ -1273,11 +1273,74 @@ function returnRandomName(character){
...
+    case character.race === "Gnome" && character.gender === "female":
+    character.firstName = utils.returnRandomArrayItem(femaleGnomeNames);
+    character.lastName = utils.returnRandomArrayItem(gnomeSurnames);
+    break;
+
+    case character.race === "Half-Elf" && character.gender === "male":
+    character.firstName = utils.returnRandomArrayItem(maleHalfElfNames);
+    character.lastName = utils.returnRandomArrayItem(halfElfSurnames);
+    break;
+    case character.race === "Hal-Elf" && character.gender === "female":
```
Comment (KingFruit85) [inline names.js]: Typo in Half-Elf

**Review by KingFruit85 — CHANGES_REQUESTED** [review]
Typo in Half-Elf on line 1303

**Review by KingFruit85 — APPROVED** [review]
👏

**Review by KingFruit85 — APPROVED** [review]
👏👏👏👏👏👏👏

File: names.js (hunk 1 above)
Comment (HedgehogHolly) [reply names.js]: Good spot! 👍

**Pull request merged**